feder/monitorings/tasks.py

```diff
@@ -9,6 +9,7 @@
 def handle_mass_assign(mass_assign):
     for case in Case.objects.filter(mass_assign=mass_assign).all():
         case.update_email()
+        case.update_is_quarantined()
```

Comment:
Czy możemy utworzyć sprawy z poprawnym "is_quarantined", zamiast aktualizować to post-factum? Myślę, że wystarczy to zaktualizować w tym miejscu https://github.com/watchdogpolska/feder/blob/215698b989b738df9721e4bc865443ccb8f902b9/feder/monitorings/views.py#L408 i nie trzeba będzie nic aktualizować dodatkowo.

Reply:
Słusznie, tak spróbuję.